tests/conftest.py

```diff
@@ -1,48 +1,70 @@
-import asyncio
-from unittest.mock import AsyncMock, MagicMock
+from unittest.mock import patch
 
+import aiohttp
 import pytest
+import pytest_asyncio
+from aioresponses import aioresponses
 
-from .aiohttp import AiohttpClientMocker
-from .mock_backendselector import BackendSelectorMock
+from tests.mock_backendselector import (
+    BackendSelectorMock,
+    BackendSelectorMockMultipleCreds,
+)
+from whirlpool.aircon import Aircon
+from whirlpool.appliancesmanager import AppliancesManager
+from whirlpool.auth import Auth
+from whirlpool.oven import Oven
+from whirlpool.washerdryer import WasherDryer
+
+SAID = "WPR1XYZABC123"
 
 
 @pytest.fixture
-def http_client_mock(mocker, backend_selector_mock):
-    """Set up aiohttp ClientSession mock."""
-    http_client_mocker = AiohttpClientMocker()
-    mocker.patch(
-        "aiohttp.ClientSession",
-        side_effect=lambda *args, **kwargs: http_client_mocker.create_session(
-            asyncio.get_event_loop()
-        ),
-    )
-    return http_client_mocker
+def aioresponses_mock():
+    with aioresponses() as m:
+        yield m
 
 
-@pytest.fixture
-def appliance_http_client_mock(
-    http_client_mock, backend_selector_mock: BackendSelectorMock
-):
-    http_client_mock.get(
-        backend_selector_mock.websocket_url, json={"url": "wss://something"}
-    )
+@pytest_asyncio.fixture(scope="session")
+async def client_session_fixture():
+    session = aiohttp.ClientSession()
+    yield session
+    await session.close()
 
-    return http_client_mock
+
+@pytest.fixture(params=[BackendSelectorMock, BackendSelectorMockMultipleCreds])
+def backend_selector_mock(request):
+    yield request.param()
 
 
 @pytest.fixture
-def backend_selector_mock():
-    return BackendSelectorMock()
+def auth_mock(backend_selector_mock, client_session_fixture):
```

Comment:
```suggestion
pytest.fixture(name="auth")
def auth_fixture(backend_selector_mock, client_session_fixture):
```
seeing "auth_mock" in the tests where Auth is being tested was getting me confused, since it is not a mock